packages/edit-navigation/package.json

```diff
@@ -20,6 +20,9 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"sideEffects": [
+		"build-style/**"
```

Comment:
Should it have:
`"!((src|build|build-module)/components/**)"` as well?

Reply:
Done, thanks! 👍